perses/rjmc/topology_proposal.py

```diff
@@ -59,14 +59,18 @@
 STRONG_ATOM_EXPRESSION = oechem.OEExprOpts_Hybridization  | oechem.OEExprOpts_HvyDegree | oechem.OEExprOpts_DefaultAtoms
 STRONG_BOND_EXPRESSION = oechem.OEExprOpts_DefaultBonds
 
+#specific to proteins
```

Comment:
there are probably better protein atom and bond expressions, but these seem to work, at least in generating topology, geometry, and hybrid system proposals that don't fail